sys/shell/commands/sc_lwip_netif.c

```diff
@@ -75,8 +91,9 @@ static void _netif_list(struct netif *netif) {
 
 #ifdef MODULE_LWIP_IPV6
     for (i = 0; i < LWIP_IPV6_NUM_ADDRESSES; i++) {
-        if (netif_ip6_addr_state(netif, i) != IP6_ADDR_INVALID) {
-            _netif_list_ipv6(netif, i);
+        uint8_t state = netif_ip6_addr_state(netif, i);
+        if (state != IP6_ADDR_INVALID) {
```

Comment:
We could do `!ip6_addr_isinvalid(state)` here to use the helpers everywhere. Either way is fine.

Reply:
👍, will do

Reply:
Done. I also added a comment to point out that `ip6_addr_isvalid()` and `!ip6_addr_isinvalid()` are not the same.